data/reusables/actions/private-repository-forks-overview.md

```diff
@@ -1 +1 @@
-If you rely on using forks of your private repositories, you can configure policies that control how users can run workflows on `pull_request` events. Available to private {% ifversion ghec or ghes or ghae %}and internal{% endif %} repositories only, you can configure these policy settings for {% ifversion ghec %}enterprises, {% elsif ghes or ghae %}your enterprise, {% endif %}organizations, or repositories.
+If you rely on using forks of your private repositories, you can configure policies that control how users can run workflows on `pull_request` events. Available to private {% ifversion ghec or ghes or ghae %}and internal{% endif %} repositories only, you can configure these policy settings for {% ifversion ghec %}enterprises, {% elsif ghes or ghae %}your enterprise, {% endif %}organizations or repositories.
```

Comment:
When versioning gets too complicated, it's okay to allow a little bit of repetition in order to make the source easier to read. 😊 
```suggestion
If you rely on using forks of your private repositories, you can configure policies that control how users can run workflows on `pull_request` events. {% ifversion fpt %}Available to private repositories only, you can configure these policy settings for organizations or repositories.{% else %}Available to private and internal repositories only, you can configure these policy settings for {% ifversion ghec %}enterprises{% else %}your enterprise{% endif %}, organizations, or repositories.{% endif %}
```